distributed/tests/test_worker_state_machine.py

```diff
@@ -1363,11 +1373,29 @@ def test_throttling_does_not_affect_first_transfer(ws):
         ComputeTaskEvent.dummy(
             "c",
             who_has={"a": [ws2]},
-            nbytes={"a": 200},
+            nbytes={"a": int(10e7)},  # transfer_message_target_bytes = 50e6 bytes
```

Comment:
Explicitly set the limit on the workerstate object in case defaults ever change?